frontend/src/services/researchGroupService.js

```diff
@@ -4,4 +4,7 @@ export default {
     fetchGroups() {
         return api.get("research-group/").then((response) => response.data);
     },
+    postGroup(data) {
+        return api.post("research-group/", data).then((response) => response.data);
+    }
```

Comment:
Tu też musi być chyba jakaś obsługa błędów, bo jak na przykład nie zrobiłem migracji i nie ma takiej tabeli to nawet nie wiem że koła nie utworzono, zobaczyłem po logach w kontenerze

Reply:
![obraz](https://user-images.githubusercontent.com/68618094/202913600-d36e8036-2f88-4335-8ee6-d25baef18e83.png)

Teraz powinno wyłapać błąd. Potem jak będzie więcej czasu można zrobić solidniejszy errorHandler